src/codeu/chat/client/commandline/Chat.java

```diff
@@ -575,10 +586,34 @@ public void invoke(List<String> args) {
         final String message = args.size() > 0 ? args.get(0) : "";
         if (message.length() > 0) {
           conversation.add(message);
+          updateInterests();
         } else {
           System.out.println("ERROR: Messages must contain text");
         }
       }
+
+      private void updateInterests(){
```

Comment:
Comment explaining the function of this method. For example, telling what type of interest is being updated.